### DISCUSSION TITLE
Cannot find /learn folder in Next.js GitHub repository

### DISCUSSION THREAD

**Discussion by IDKNWHORU** [vercel/next.js#68791] [Help] [answered]
### Summary

I hope this message finds you well. I'm writing to inquire about a difficulty I've encountered while working on the Korean translation of Next.js documentation. Specifically, I'm unable to locate the /learn folder in the Next.js GitHub repository, which I need to access the "Learn React Foundation" and "Dashboard App" content mentioned in the Pre-Requisite Knowledge section.

I've searched thoroughly in the repository but haven't been able to find the /learn folder or these resources. As I'm working on translating these materials into Korean, it's crucial for me to access the original content.

Could you kindly provide guidance on where I might find these materials or if they have been moved or renamed?

Your assistance in this matter would be greatly appreciated, as it will help me continue the translation work and contribute to making Next.js more accessible to Korean-speaking developers.

Thank you for your time and consideration. I look forward to your response.

### Additional information

```js
## Pre-Requisite Knowledge

Although our docs are designed to be beginner-friendly, we need to establish a baseline so that the docs can stay focused on Next.js functionality. We'll make sure to provide links to relevant documentation whenever we introduce a new concept.

To get the most out of our docs, it's recommended that you have a basic understanding of HTML, CSS, and React. If you need to brush up on your React skills, check out our [React Foundations Course](/learn/react-foundations), which will introduce you to the fundamentals. Then, learn more about Next.js by [building a dashboard application](/learn/dashboard-app).
```

### Example

https://github.com/vercel/next.js/blob/v14.2.5/docs/index.mdx?plain=1

**Comment by icyJoseph** [comment, marked as answer]
I think some of the learn material is not yet open source, or might never be. 

Here, https://github.com/vercel/next-learn you can find some of the **_code_** for the examples in the learning material. I think you can open an issue there, and ask more about how to be able to contribute with translations.

**Reply by IDKNWHORU** [reply]
thank you Let me ask them.
